source/mouseHandler.py

```diff
@@ -201,19 +201,18 @@ def getTotalWidthAndHeightAndMinimumPosition(displays):
 	return (totalWidth, totalHeight, wx.Point(smallestX, smallestY))
 
 def executeMouseMoveEvent(x,y):
-	global currentMouseWindow
 	desktopObject=api.getDesktopObject()
 	displays = [ wx.Display(i).GetGeometry() for i in range(wx.Display.GetCount()) ]
 	x, y = getMouseRestrictedToScreens(x, y, displays)
 	screenWidth, screenHeight, minPos = getTotalWidthAndHeightAndMinimumPosition(displays)
+	oldMouseObject = api.getMouseObject()
+	mouseObject = desktopObject.objectFromPoint(x, y)
 
-	if config.conf["mouse"]["audioCoordinatesOnMouseMove"]:
+	if config.conf["mouse"]["audioCoordinatesOnMouseMove"] and not oldMouseObject.appModule.sleepMode:

```

Comment:
should this use the new mouse object instead?

```suggestion
	if config.conf["mouse"]["audioCoordinatesOnMouseMove"] and not mouseObject.appModule.sleepMode:
```

Reply:
I was considering the possibility of failure in the logic of updating the mouse object.

So to be on the safe side I chose to use the old mouse object for the judgment.

Although this will delay the response of a mouse object, I think it is acceptable for this PR.